src/pages/resultsView/mutation/ResultsViewMutationTable.tsx

```diff
@@ -40,19 +48,62 @@ export default class ResultsViewMutationTable extends MutationTable<IResultsView
             MutationTableColumnType.MUTATION_ASSESSOR,
             MutationTableColumnType.COSMIC,
             MutationTableColumnType.TUMOR_ALLELE_FREQ,
-            MutationTableColumnType.NORMAL_ALLELE_FREQ
+            MutationTableColumnType.NORMAL_ALLELE_FREQ,
+            MutationTableColumnType.CANCER_TYPE,
+            MutationTableColumnType.NUM_MUTATIONS
```

Comment:
These new columns (`CancerType` and `NumMutations`) can be moved to the base class if we like. Seems like they don't have special dependencies to any other component. But we can leave them here too if we think we don't have any general use case for these columns.